aplijobs/birdwoot
# Feature || Change chat window favicon

# Pull Request Template

## Description

Change chat window favicon

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Run birdwoot
- Start a conversation
- Check APLI's favicon is now on the tab

<details>
<summary>New favicon</summary>

![image](https://user-images.githubusercontent.com/9289461/223722280-01477deb-255b-4ed8-adbe-de90941a9693.png)


</details>
